### PR TITLE
load testing resources from classpath path

### PR DESCRIPTION
**Pull request by mwiede**
I was importing the test-jar of com.github.docker-java:docker-java to reuse the test classes. I found the problem, that this was not possible until the files used in testing are loading from classpath instead of source path.

### CODE REVIEW

**Comment by mwiede** [comment]
@eddumelendez could you please consider reopen and merge this? Thanks

File: docker-java/src/test/java/com/github/dockerjava/cmd/CopyArchiveToContainerCmdIT.java [hunk 1]
```diff
@@ -28,12 +28,12 @@ public class CopyArchiveToContainerCmdIT extends CmdIT {
...
-        CompressArchiveUtil.tar(Paths.get("src/test/resources/testReadFile"), temp, true, false);
+        CompressArchiveUtil.tar(Paths.get(ClassLoader.getSystemResource("testReadFile").toURI()), temp, true, false);
         try (InputStream uploadStream = Files.newInputStream(temp)) {
             dockerRule.getClient()
-                    .copyArchiveToContainerCmd(container.getId())
-                    .withTarInputStream(uploadStream)
-                    .exec();
+                .copyArchiveToContainerCmd(container.getId())
+                .withTarInputStream(uploadStream)
+                .exec();
```
Comment (eddumelendez) [inline docker-java/src/test/java/com/github/dockerjava/cmd/CopyArchiveToContainerCmdIT.java:36]: Can we avoid those changes, please?

Comment (mwiede) [reply docker-java/src/test/java/com/github/dockerjava/cmd/CopyArchiveToContainerCmdIT.java]: ok reverted thosw whitespace changes

**Review by eddumelendez — APPROVED** [review]

**Comment by eddumelendez** [comment]
Thanks for your contribution, @mwiede ! and sorry about the delay.

**Pull request merged**